src/data-workspace/validation/validation-comments-violations.js

```diff
@@ -34,7 +34,10 @@ const ValidationCommentsViolations = ({ commentRequiredViolations }) => {
                             >
                                 {i18n.t(
                                     "{{displayShortName}} must have a comment when it doesn't have a value.",
-                                    { displayShortName }
+                                    {
+                                        displayShortName,
+                                        interpolation: { escapeValue: false },
```

Comment:
this is consistent with decisions elsewhere (deem react sanitization sufficient and turn off i18next's escaping to make sure characters are printed correctly)